src/docs/product/issues/issue-details/performance-issues/n-one-queries.mdx

```diff
@@ -6,7 +6,7 @@ redirect_from:
 description: "Learn more about N+1 queries and how to diagnose and fix them."
 ---
 
-_N+1 queries_ are a performance problem in which the application makes database queries in a loop, instead of making a single query that returns all the information at once. Each database connection takes some amount of time, so querying the database in a loop can be many times slower than doing it just once. This problem often occurs when you use an object-relational mapping (ORM) tool in web frameworks like Django or Ruby on Rails.
+_N+1 queries_ are a performance problem in which the application makes database queries in a loop, instead of making a single query that returns all the information or makes many modifications at once. Each database connection takes some amount of time, so querying the database in a loop can be many times slower than doing it just once. This problem often occurs when you use an object-relational mapping (ORM) tool in web frameworks like Django or Ruby on Rails.
```

Comment:
```suggestion
_N+1 queries_ are a performance problem in which the application makes database queries in a loop, instead of making a single query that returns or modifies all the information at once. Each database connection takes some amount of time, so querying the database in a loop can be many times slower than doing it just once. This problem often occurs when you use an object-relational mapping (ORM) tool in web frameworks like Django or Ruby on Rails.
```